chart/values.schema.json

```
@@ -1450,6 +1450,14 @@
                         "type": "string"
                     }
                 },
+                "labels": {
+                    "description": "Labels to add to the worker pods.",
```

```suggestion
                    "description": "Labels to add to the worker objects and pods.",
```

We apply these to all worker objects, not just the pod. This matches what we have for `labels` currently.